gateway/src/test/java/io/zeebe/gateway/broker/BrokerClientTest.java

```diff
@@ -203,31 +154,6 @@ public void shouldThrowExceptionIfPartitionNotFoundResponse() {
     client.sendRequest(new BrokerCreateWorkflowInstanceRequest()).join();
   }
 
-  @Test
-  public void shouldEstablishNewConnectionsAfterDisconnect() {
```

Comment:
This test is still valid and should not be removed, instead of the topology request we should use another request i.e. deployment or similar.